adafruit_midi/__init__.py

```diff
@@ -26,7 +26,7 @@
 A CircuitPython helper for encoding/decoding MIDI packets over a MIDI or UART connection.
```

Comment:
_(For Line 23 since GitHub won't let me comment on that line)_
For Sphinx `autodoc` this needs to match the package naming. I think this one should be `adafruit_midi.adafruit_midi`.